src/Htmxor/Http/HtmxResponse.cs

```diff
@@ -135,8 +135,9 @@ public HtmxResponse ReplaceUrl(string url)
     /// Allows you to specify how the response will be swapped.
     /// </summary>
     /// <param name="swapStyle"></param>
+	/// <param name="modifier">The hx-swap attributes supports modifiers for changing the behavior of the swap.</param>
     /// <returns>This <see cref="HtmxResponse"/> object instance.</returns>
-    public HtmxResponse Reswap(SwapStyle swapStyle)
+    public HtmxResponse Reswap(SwapStyle swapStyle, string? modifier = null)
```

Comment:
Would this be compatible with `hx-swap="show:none"`?

Reply:
Good catch. Took a dive into the htmx source.  It seems like modifiers can be used without necessarily specifying a swap style.  The swap specification is loaded over top of the defaults.  It defaults swapStyle to innerHTML if boosted, otherwise whatever the configured defaultSwapStyle is.  Then reswap can come in and change the swapstyle and/or add modifiers.  Modifiers can be used independently of Swapstyle.  

I'm going to overload Reswap with just a string parameter to allow modifiers only.